java/src/main/java/com/xebia/functional/xef/java/auto/AIScope.java

```diff
@@ -99,14 +108,17 @@ public CompletableFuture<List<String>> promptMessages(Chat llmModel, String prom
         return exec.future(continuation -> scope.promptMessages(llmModel, prompt, functions, promptConfiguration, continuation));
     }
 
+    public Publisher<String> promptStreaming(Chat gpt4all, String line, PromptConfiguration promptConfiguration) {
+        return ReactiveFlowKt.asPublisher(scope.promptStreaming(gpt4all, line, exec.getContext(), null, Collections.emptyList(), promptConfiguration));
```

Comment:
```suggestion
        return ReactiveFlowKt.asPublisher(scope.promptStreaming(gpt4all, line, exec.getContext(), scope.conversationId, Collections.emptyList(), promptConfiguration));
```
This should be the current scope conversation id so the memory and previous messages are included in the prompt